### PR TITLE
chore(releasing): Prepare v0.25.2 release

### PR DESCRIPTION
**Pull request by spencergilbert**

### CODE REVIEW

**Comment by neuronull** [comment]
I think the CI check might be failing because we didn't add a 0.25.2 git tag.

File: website/cue/reference/releases/0.23.0.cue [hunk 1]
```diff
@@ -26,6 +26,7 @@ releases: "0.23.0": {
 	known_issues: [
 		"Vector shuts down when a configured source codec (`decoding.codec`) receives invalid data. Fixed in v0.23.1.",
 		"The `elasticsearch` sink doesn't evaluate templated configuration options like the `index` configuration before applying the `only_fields` and `except_fields` options, causing templates to fail to be evaluated if they used a field that was dropped. Fixed in v0.23.1.",
+		"The `datadog_traces` sink APM stats calculation does not correctly aggregate the stats in the way that is expected by the APM backend of Datadog, causing incorrect individual span metrics observed in the Datadog UI. Fixed in v0.25.2."
```
Comment (archoversight) [inline website/cue/reference/releases/0.23.0.cue:29]:
```suggestion
		"The `datadog_traces` sink APM stats calculation does not correctly aggregate the stats in the way that is expected by the APM backend of Datadog, causing incorrect individual span metrics observed in the Datadog UI. Fixed in v0.25.2.",
```

As mentioned in the docs-check: https://github.com/vectordotdev/vector/actions/runs/3567742513/jobs/5995805639#step:14:31

File: website/cue/reference/releases/0.25.2.cue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+			scopes: ["datadog_traces sink"]
+			description: """
+				The `datadog traces` sink now correctly aggregates (caches) the computed APM stats, and sends the aggregated
+				payloads to Datadog at a fixed interval, decoupled from the trace payloads. The stats payloads were previously
+				not being cached and instead sent to Datadog as each batch of traces was processed, which is not compatible
+				with Datadog's backend APM API. A robust integration test was also added, to validate the correct aggregation
+				behavior.
+				"""
+			pr_numbers: [14694, 14757, 14861, 15084]
+		}
```
Comment (archoversight) [inline website/cue/reference/releases/0.25.2.cue:45]:
```suggestion
		},
```

Suggested by: https://github.com/vectordotdev/vector/actions/runs/3567742513/jobs/5995805639#step:14:40

**Review by archoversight — COMMENTED** [review]
This release fixes a real issue with Azure for us, and I am looking forward to getting it deployed.

**Review by jszwedko — APPROVED** [review]

**Comment by jszwedko** [comment]
Merged locally to squash release prep commits and avoid squashing the cherry-picked commits.

**Pull request closed without merging**